packages/rabbitmq/src/TxSubmitWorker.ts

```diff
@@ -0,0 +1,182 @@
+import { Channel, Connection, Message, connect } from 'amqplib';
+import { TxSubmissionQueue } from './rabbitmqTxSubmitProvider';
+import { TxSubmitProvider } from '@cardano-sdk/core';
+import { ogmiosTxSubmitProvider, urlToConnectionConfig } from '@cardano-sdk/ogmios';
```

Comment:
This is bound to [previous discussion](https://github.com/input-output-hk/cardano-js-sdk/pull/254#discussion_r877717364). I'll change the interface in order to accept an already instantiated `TxSubmitProvider` rather than an ogmios URL (from which ATM I'm instantiating the `TxSubmitProvider`).

Reply:
I forgot to mention it would be good to follow this pattern of typing the dependencies: https://github.com/input-output-hk/cardano-js-sdk/blob/4265f6af60a92c93604b93167fd297530b6e01f8/packages/cardano-services/src/StakePoolSearch/StakePoolSearchHttpService.ts#L11-L14